troubleshooting/tools/tracevol.py

```diff
@@ -185,7 +185,7 @@ def check_pv_name_in_rados(arg, image_id, pvc_name, pool_name, is_rbd):
     """
     validate pvc information in rados
     """
-    omapkey = 'csi.volume.%s' % pvc_name
+    omapkey = 'csi.volume.%s'.format(pvc_name)
```

Comment:
@Madhu-1 
```suggestion
    omapkey = f'csi.volume.{pvc_name}'
```

or
```suggestion
    omapkey = 'csi.volume.{}'.format(pvc_name)
```

from https://miguendes.me/pylint-consider-using-f-string

Reply:
:thinking: , maybe try 

```suggestion
    omapkey = f'csi.volume.{pvc_name}'
```
I am not able to reproduce locally too.